components/clp-package-utils/clp_package_utils/scripts/native/decompress.py

```diff
@@ -93,16 +104,6 @@ def main(argv):
     )
     parsed_args = args_parser.parse_args(argv[1:])
 
-    # Validate paths were specified using only one method
-    if len(parsed_args.paths) > 0 and parsed_args.files_from is not None:
-        args_parser.error("Paths cannot be specified both on the command line and through a file.")
-
-    # Validate extraction directory
-    extraction_dir = pathlib.Path(parsed_args.extraction_dir)
-    if not extraction_dir.is_dir():
-        logger.error(f"extraction-dir ({extraction_dir}) is not a valid directory.")
-        return -1
-
     # Validate and load config file
```

Comment:
Can we move this into a method and then call it after validating the arguments? I think it's more intuitive to validate the arguments first, then the config file.

Reply:
like `validate_decompression_(command)_arguments`?

Reply:
Sorry, I mean can we move the `Validate and load config file` block.